src/main/java/org/jabref/gui/preferences/journals/AbbreviationsFileViewModel.java

```diff
@@ -35,6 +35,7 @@ public AbbreviationsFileViewModel(Path filePath) {
         this.path = Optional.ofNullable(filePath);
         this.name = path.get().toAbsolutePath().toString();
         this.isBuiltInList = new SimpleBooleanProperty(false);
+        this.abbreviations.add(new AbbreviationViewModel(null));
```

Comment:
This is a so called PseudoAbbreviation. I have not really understood what the real benefit is of this (maybe one can doubleclick an empty space in the List and insert a new abbreviation?) It's buggy, makes the code quite complicated to understand and I see no real benefit in it. ... ... But I put it back in for now, as several tests break, because they expect this PseudoAbbreviation and it would take me now valuable life time I would rather like to spend in finishing this PR, maybe as soon as I have completed everything else...